doc/guide.rst

```diff
@@ -201,6 +201,21 @@ What this does is tell pyOptSparse that the key ``obj_name`` in the function ret
 For optimizers that can do multi-objective optimization (e.g. NSGA2), multiple objectives can be added.
 Optimizers that can only handle one objective enforce that only a single objective is added to the optimization description.
 
+Analytic Derivatives
+++++++++++++++++++++
+
+pyOptSparse can automatically compute derivatives of the objective and constraint functions using finite differences or the complex-step method (See implementation details `here <https://mdolab-pyoptsparse.readthedocs-hosted.com/en/latest/api/gradient.html>`__).
+If analytic derivatives are available, users can define them within the ``sens()`` function.
```

Comment:
I think `sens()` can have any name, it's the `sens=` key in the `Optimizer` instance that has to be used, as you point out a few lines later. Just specify that `sens` is only for explanation purposes here.

Reply:
Good point. I will update this.